src/paradigma/config.py

```diff
@@ -0,0 +1,408 @@
+        self.fmin_low_power: float = 0.5
+        self.fmax_low_power: float = 3
+
+        # cepstral coefficients
+        self.segment_length_s_mfcc: float = 2
+        self.fmin_mfcc: float = 0
+        self.fmax_mfcc: float = 25
+        self.n_dct_filters_mfcc: int = 15
+        self.n_coefficients_mfcc: int = 12
+
+        self.d_channels_values: Dict[str, str] = {}
+        for mfcc_coef in range(1, self.n_coefficients_mfcc + 1):
+            self.d_channels_values[f"mfcc_{mfcc_coef}"] = "unitless"
+
+        self.d_channels_values["freq_peak"] = "Hz"
+        self.d_channels_values["low_freq_power"] = "(deg/s)^2"
+        self.d_channels_values["tremor_power"] = "(deg/s)^2"
+
+
+class SignalQualityFeatureExtractionConfig(HeartRateBaseConfig):
+
+    def __init__(self) -> None:
+        super().__init__()
+
+        self.window_length_welch = 3 * self.sampling_frequency
+        self.overlap_welch_window = self.window_length_welch // 2
+
+        self.freq_band_physio = [0.75, 3] # Hz
+        self.bandwidth = 0.2   # Hz
+
+        config_imu = IMUConfig()
+        self.sampling_frequency_imu = config_imu.sampling_frequency
+
+        self.single_value_cols: List[str] = None
+        self.list_value_cols: List[str] = [
+            self.ppg_colname
+        ]
+
+
+# Classification
+class GaitDetectionConfig(GaitBaseConfig):
+
+    def __init__(self) -> None:
+        super().__init__()
+
+        self.classifier_file_name = "gait_detection_classifier.pkl"
+        self.thresholds_file_name = "gait_detection_threshold.txt"
+
+        self.set_filenames('gait')
+
+
+class FilteringGaitConfig(GaitBaseConfig):
+
+    def __init__(self) -> None:
+        super().__init__()
+
+        self.classifier_file_name = "gait_filtering_classifier.pkl"
+        self.thresholds_file_name = "gait_filtering_threshold.txt"
+
+        self.set_filenames('arm_activity')
+
+
+class TremorDetectionConfig(TremorBaseConfig):
+
+    def __init__(self) -> None:
+        super().__init__()
+
+        self.coefficients_file_name = "tremor_detection_coefficients.txt"
+        self.thresholds_file_name = "tremor_detection_threshold.txt"
+        self.mean_scaling_file_name = "tremor_detection_mean_scaling.txt"
+        self.std_scaling_file_name = "tremor_detection_std_scaling.txt"
+
+        self.fmin_peak: float = self.fmin_tremor_power
+        self.fmax_peak: float = self.fmax_tremor_power
+
+        self.d_channels_values = {
+            DataColumns.PRED_TREMOR_PROBA: "probability",
+            DataColumns.PRED_TREMOR_LOGREG: "boolean",
+            DataColumns.PRED_TREMOR_CHECKED: "boolean"
+        }
+
+        self.set_filenames('tremor')
+
+
+class SignalQualityClassificationConfig(HeartRateBaseConfig):
+
+    def __init__(self) -> None:
+        super().__init__()
+        self.classifier_file_name = "ppg_quality_classifier.pkl"
+        self.thresholds_file_name = "ppg_acc_quality_threshold.txt"
+
+        self.set_filenames('gait')
```

Comment:
ppg

Reply:
@KarsVeldkamp er is nu gemerged maar deze staat nog. Kan jij deze meenemen in je volgende PR?